test/torchaudio_unittest/example/emformer_rnnt/test_utils.py

```diff
@@ -0,0 +1,32 @@
+class MockSentencePieceProcessor:
```

Comment:
The file name for utilities cannot include `test` prefix / suffix. IIRC, it causes some issue with pytest.